src/components/Editor/SearchBar.js

```diff
@@ -399,12 +402,15 @@ const SearchBar = React.createClass({
   },
 
   // Handlers
-  selectResultItem(item: FormattedSymbolDeclaration) {
+  selectResultItem(e: SyntheticEvent, item: SymbolDeclaration, index?: number) {
```

Comment:
Hmm, my only question is around `FormattedSymbolDeclaration` => `SymbolDeclaration`. Why was this needed?